src/ghga_transpiler/workbook_parser.py

```diff
@@ -112,7 +112,14 @@ def _relations(self, row: dict) -> dict:
         resource that is in the relation as the value
         """
         relations = self.config.get_relations()
-        return {relation: row[relation] for relation in relations if relation in row}
+        return {
+            relation[0]: {
+                "targetClass": relation[1],
+                "targetResources": row[relation[0]],
+            }
+            for relation in relations
+            if relation[0] in row
+        }
```

Comment:
For the sake of readability, and actually understanding what's going on here, this should either be commented or, better, `self.config.get_relations` should return something like a `namedtuple` or `dataclass` that replaces the indices wit a named field.